### DISCUSSION TITLE
"columns" mode not working properly

### DISCUSSION THREAD

**Discussion by flawnn** [lowlighter/metrics#699] [Q&A] [answered]
Hello everyone!
When using the "columns" displaymode the metrics are rendered buggy and not working properly on my profile. The actual SVG is afaik without any problem. I guess that's nothing you can do about right now but what about adding a new approach to this? I find it much more aesthetically widening the metrics out to the full width of that README block in the profile, what do you all think?

**Comment by lowlighter** [comment, marked as answer]
Hello!
Yes it's more or less related to #374, there are some features not available when using svg inside `<img/>` tags which cause some issues because the columns display mode rely on media queries, and iirc the `width` attribute needs to be set on the html tag itself in order to work

Unfortunately no one came up with a solution yet 😕 

There's always the workaround of splitting your metrics in 2 and using `align="left"` and `align="right"` but it won't be exactly the same display indeed

**Reply by lowlighter** [reply]
Linking https://github.com/lowlighter/metrics/issues/839 and archiving 👍
